src/db/spanner/models.rs

```diff
@@ -558,11 +561,17 @@ impl SpannerDb {
     }
 
     pub fn delete_storage_sync(&self, user_id: params::DeleteStorage) -> Result<()> {
-        // XXX: should delete from bso table too
         let user_id = user_id.legacy_id as u32;
         self.sql("DELETE FROM user_collections WHERE userid=@userid")?
             .params(params! {"userid" => user_id.to_string()})
             .execute(&self.conn)?;
+        self.sql("DELETE FROM bso WHERE userid=@userid")?
+            .params(params! {"userid" => user_id.to_string()})
+            .execute(&self.conn)?;
+        // Erect a Tombstone
+        self.sql("INSERT INTO user_collections (userid, collection, last_modified) values (@userid, \"0\", @modified)")?
```

Comment:
We need to track collection deletion (delete_collection_sync) whereas this (delete_storage_sync) is tracking user deletion.

Reply:
Ah, I misunderstood the requirement. If we're noting that a given user record is modified by the removal of collections, should we keep a tombstone when all collections are deleted as well, or if everything is deleted does it not really matter?

Reply:
It doesn't matter when everything's deleted. storage's last_modified in that case resets to 0